### PR TITLE
added View App button to POS settings

### PR DESCRIPTION
**Pull request by nakamotones**

### CODE REVIEW

File: BTCPayServer/Models/AppViewModels/UpdatePointOfSaleViewModel.cs [hunk 1]
```diff
@@ -60,6 +60,9 @@ public class UpdatePointOfSaleViewModel
         [Display(Name = "Redirect invoice to redirect url automatically after paid")]
         public string RedirectAutomatically { get; set; } = string.Empty;
 
+        public string AppId { get; set; }
```
Comment (NicolasDorier) [inline BTCPayServer/Models/AppViewModels/UpdatePointOfSaleViewModel.cs]: where is it used?

Comment (nakamotones) [reply BTCPayServer/Models/AppViewModels/UpdatePointOfSaleViewModel.cs]: Used in UpdatePointOfSale.cshtml. Build fails without it.

**Comment by rockstardev** [comment]
Looking good, I'll resolve that issue with Azure test.

**Comment by Kukks** [comment]
I fixed the test issue on your pull request directly. The new things failing are your wallet changes

**Comment by Kukks** [comment]
I would really prefer having the wallets changes as a separate pull request as it's a whole new section up for discussion ;)

**Comment by radWorx** [comment]
Ah, I didn't realize it attached to the original pr.

**Comment by NicolasDorier** [comment]
You need to rebase on master.

**Comment by NicolasDorier** [comment]
Conflict again, need rebase.

**Comment by nakamotones** [comment]
Will do.

**Pull request closed without merging**